Realm/RLMSyncUtil.mm

```diff
@@ -45,8 +45,12 @@
 
 using namespace realm;
 
-static_assert((int)RLMClientResetModeManual == (int)realm::ClientResyncMode::Manual);
```

Comment:
Can we convert RLMClientResetModeManual to RLMClientResetModeDiscardUnsyncedChanges, so we don't have this warning on the code

Reply:
RLMClientResetModeManual will still need to assert to manual. For RLMClientResetModeDiscardLocal do you mean like convert in our objc convert file?